portal/static/js/main.js

```diff
@@ -2271,7 +2263,7 @@ var Global = {
                         addUserId = "&user_id=" + $el.attr("data-user-id");
                     }
                     if (emailReg.test(emailVal)) {  // If this is a valid address, then use unique_email to check whether it's already in use
-                        tnthAjax.sendRequest("/api/unique_email?email=" + encodeURIComponent(emailVal) + addUserId, "GET", "", null, function(data) {
+                        tnthAjax.sendRequest("/api/unique_email?email=" + encodeURIComponent(emailVal.toLowerCase()) + addUserId, "GET", "", null, function(data) {
```

Comment:
I'm not confident that [email addresses are entirely case-insensitive](https://stackoverflow.com/a/9808332), and even so I think this should probably be implemented in the `unique_email` API.

Could you open a new PR for this either way?